crates/objc2-encode/src/lib.rs

```diff
@@ -55,8 +55,7 @@ mod encoding_box;
 mod helper;
 mod parse;
 
-// Will be used at some point when generic constants are available
```

Comment:
Nit: I'd like to keep this comment, as that is still what the purpose of that module is.